src/mainwin.cc

```diff
@@ -732,8 +733,9 @@ void MainWindow::updateReorderMode(bool reorderMode)
 
 MainWindow::~MainWindow()
 {
-	if (root_module) delete root_module;
-	if (root_node) delete root_node;
+	delete root_module;
+	delete last_good_module;
```

Comment:
My change shouldn't change behavior; deleting an object doesn't modify the pointer. This was merely a coding style update.